build.gradle

```diff
@@ -44,6 +44,8 @@ dependencies {
     compile("com.fasterxml.jackson.core:jackson-databind:2.4.3")
     compile("org.springframework.data:spring-data-redis:1.5.0.RELEASE")
     compile("javax.servlet:jstl:1.2")
+    compile("io.springfox:springfox-swagger2:2.4.0")
+    compile("io.springfox:springfox-swagger-ui:2.4.0")
```

Comment:
is there a way we can only build this for dev environments so we don't do this on every deploy where we don't need the docs?